lib/utils/basic.d.ts

```
@@ -2,16 +2,29 @@ import type { ComponentPublicInstance } from 'vue';
+    <T extends {}, U, V>(target: T, source1: U, source2: V): T & U & V;
+    <T extends {}, U, V, W>(target: T, source1: U, source2: V, source3: W): T & U & V & W;
     (target: object, ...sources: any[]): any;
 };
 export declare const inBrowser: boolean;
-export declare type Numeric = number | string;
-export declare type ComponentInstance = ComponentPublicInstance<{}, any>;
+export type Numeric = number | string;
+export type ComponentInstance = ComponentPublicInstance<{}, any>;
+export declare const isObject: (val: unknown) => val is Record<any, any>;
+export declare const isDef: <T>(val: T) => val is NonNullable<T>;
+export declare const isFunction: (val: unknown) => val is Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)